.github/scripts/check-folders.js

```diff
@@ -18,6 +18,8 @@ async function main() {
         'platforms-serverless/firebase-functions/functions', // Firebase root doesn't have package.json but is included
         'generic/basic', // generic/basic doesn't use Github action matrix feature which we parse to find out the differences
         'packagers/yarn-workspaces/prisma-project', // Yarn workspaces root doesn't have package.json but is included
+        'packagers/yarn3-workspaces-pnp/sub-project-1',
+        'packagers/yarn3-workspaces-pnp/sub-project-2',
```

Comment:
Looks like you could change to
```suggestion
        'packagers/yarn3-workspaces-pnp/packages',
```

Reply:
Ah thanks, I thought I was going mad.